ArjunaJTA/jms/src/main/java/org/jboss/narayana/jta/jms/ConnectionClosingSynchronization.java

```diff
@@ -43,6 +48,11 @@ public void beforeCompletion() {
         // Nothing to do
     }
 
+    /**
+     * Close the connection despite the status of the transaction.
```

Comment:
Instead of using the word despite I would use any of the following:
"Close the connection."
"Close the connection regardless of what the status of the transaction is."
"Close the connection no matter what the status of the transaction is."

Similarly for other usages of the word despite below.